src/dbus/client/thread_api_dbus.hpp

```diff
@@ -84,21 +84,21 @@ class ThreadApiDBus
      * @param[in]   aSeconds  The timeout to close the port, 0 for never close.
      *
      * @retval OT_ERROR_NONE successfully performed the dbus function call
-     * @returns Error value otherwise
+     * @retval Error value otherwise
```

Comment:
We should avoid using `@retval` like this. It doesn't work well with auto-generated documentation. See example: https://openthread.io/reference/group/api-thread-general#otthreadgetdevicerole

Prefer using `@returns` if we are not explicitly listing each possible return value.